migrations/20200420215640-remove-clinical-info-stats.js

```diff
@@ -0,0 +1,19 @@
+export async function up(db) {
```

Comment:
the dbs got wiped .. it's good that you prepared this but I don't think it's is necessary now

Reply:
Ah, was not aware. In that case I'll remove this migration script :+1: